content/ko/docs/topics/library_charts.md

```diff
@@ -0,0 +1,386 @@
+---
+title: "라이브러리 차트"
+description: "Explains library charts and examples of usage"
+aliases: ["docs/library_charts/"]
```

Comment:
Please remove this line. This will incorrectly redirect a past English page to this replacement link.

Reply:
Thanks for you comment. I will deleted this line.